super_csv/serializers.py

```diff
@@ -38,5 +40,15 @@ def get_related_queryset(cls, queryset):
         return queryset.select_related('user')
 
     def get_data(self, operation):
-        data = json.load(operation.data)
-        return CSVOperationDataSerializer(data).data
+        if bool(operation.data):
```

Comment:
This can probably be an implicit check
```suggestion
        if operation.data:
```

Reply:
That would not work https://stackoverflow.com/questions/8850415/django-filefield-check-if-none/8850547